app/models/queue_tab.rb

```diff
@@ -77,7 +77,7 @@ def on_hold_tasks
   end
 
   def recently_closed_tasks
```

Comment:
Name update to match new scope? I think it's just referenced in two spots

```suggestion
  def recently_completed_tasks
```

https://github.com/department-of-veterans-affairs/caseflow/blob/3f754ec987dabcc701f7fd103c50cb2ee883f9a3/app/models/queue_tabs/completed_tasks_tab.rb#L18-L20

https://github.com/department-of-veterans-affairs/caseflow/blob/3f754ec987dabcc701f7fd103c50cb2ee883f9a3/app/models/queue_tabs/organization_completed_tasks_tab.rb#L18-L20

Reply:
Nice catch!